## Урок 2. Дискретные случайные величины. Закон распределения вероятностей. Биномиальный закон распределения. Распределение Пуассона

#### 1. Вероятность того, что стрелок попадет в мишень, выстрелив один раз, равна 0.8. Стрелок выстрелил 100 раз. <br>Найдите вероятность того, что стрелок попадет в цель ровно 85 раз.

In [8]:
from math import factorial
factorial(100) / factorial(15) / factorial(85)  * (0.8**85) * (0.2**15) * 100

4.8061793700746565

##### Ответ: <br>вероятность попасть ровно 85 раз: ~ 4.81 %

#### 2. Вероятность того, что лампочка перегорит в течение первого дня эксплуатации, равна 0.0004. <br>В жилом комплексе после ремонта в один день включили 5000 новых лампочек. <ol><li>Какова вероятность, что ни одна из них не перегорит в первый день?</li> <li> Какова вероятность, что перегорят ровно две?</li></ol>

In [15]:
none_of_them = 0.9996**5000 * 100
two_of_them = 2500 * 4999 * (0.9996**4998) * (0.0004**2) * 100
none_of_them, two_of_them

(13.528114551440707, 27.07247150266753)

##### Ответ: <br>ни одна не перегорит ~ 13.53 % <br> ровно 2 перегорят ~ 27.07 %

#### 3. Монету подбросили 144 раза. <br>Какова вероятность, что орел выпадет ровно 70 раз?

In [18]:
factorial(144) / factorial(70) / factorial(74) * (1/2**144) * 100

6.281178035144776

##### Ответ: <br>вероятность выпадания орла ровно 70 раз: ~ 6.28 %

#### 4.В первом ящике находится 10 мячей, из которых 7 - белые. Во втором ящике - 11 мячей, из которых 9 белых. <br>Из каждого ящика вытаскивают случайным образом по два мяча. <ol><li>Какова вероятность того, что все мячи белые? </li><li>Какова вероятность того, что ровно два мяча белые? </li><li>Какова вероятность того, что хотя бы один мяч белый?</li></ol>

In [23]:
all_white = (7/10) * (6/9) * (9/11) * (8/10) * 100
# 3 возможных сценария для вытаскивания 2 белых шаров: 1я - 2, 2я - 0 ; 1я - 1, 2я - 1 ; 1я - 0, 2я - 2
two_whites = ((3/10) * (2/9) * (9/11) * (8/10) + 2 * (7/10) * (3/9) * 2 * (9/11) * (2/10) + (7/10) * (6/9) * (2/11) * (1/10)) * 100
at_least_one_white = (1 - (3/10) * (2/9) * (2/11) * (1/10)) * 100
all_white, two_whites, at_least_one_white

(30.545454545454543, 20.484848484848484, 99.87878787878788)

##### Ответ: <br>все белые ~ 30.55 % <br> ровно 2 белых ~ 20.48 %  <br> хотя бы 1 белый ~ 99.88 %